kof-operator/internal/controller/clusterdeployment_kof_cluster_role.go

```diff
@@ -115,18 +120,21 @@ func (r *ClusterDeploymentReconciler) reconcileChildClusterRole(
 		return err
 	}
 
+	if err := r.createProfile(childClusterDeployment, regionalClusterDeployment, ctx); err != nil {
```

Comment:
https://pkg.go.dev/context says "The Context should be the first parameter, typically named ctx:"
```suggestion
	if err := r.createProfile(ctx, childClusterDeployment, regionalClusterDeployment); err != nil {
```